.make/Makefile.build

```diff
@@ -12,19 +12,15 @@ tools: ## Installs required go tools
 	@go get -u github.com/jteeuwen/go-bindata/...
 
 .PHONY: install
-install: generate ## Fetches all dependencies using Glide
-	glide install -v
+install: generate ## Fetches all dependencies using dep
+	dep ensure -v
 
-.PHONY: up
-up: generate ## Updates all dependencies defined for glide
-	glide up -v
-
-.PHONY: up-skip-test-quick
-up-skip-test-quick:
-	glide update --skip-test --quick ## Updates only things in glide.yaml and skips tests
+.PHONY: ensure
+ensure: generate ## Updates all dependencies defined for dep
+	dep ensure -update -v
```

Comment:
Maybe we call it `update` instead? both `dep` targets are using `ensure`, so I was thinking maybe it would be better to name the targets closer to what they do?

Reply:
Good point - I'll rename it